.github/workflows/mac.yml

```diff
@@ -83,7 +83,11 @@ jobs:
         run: |
           python3 ./mach build --${{ inputs.profile }}
       - name: Smoketest
-        run: python3 ./mach smoketest --${{ inputs.profile }}
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 5
+          max_attempts: 3
+          command: python3 ./mach smoketest --${{ inputs.profile }}
```

Comment:
Can you please open a bug for this failure and the one for Linux and leave the bug URL in the comments?